### PR TITLE
Script to find malformed urls

### PR DESCRIPTION
**Pull request by amdomanska**
Connects to: https://github.com/DOAJ/doajPM/issues/2038
Solution for: https://github.com/DOAJ/doajPM/issues/2038#issuecomment-518259336 (p. 2)

In the portality/scripts you can find articles_invalid_url.py script. Running that on our live server will give us info about which articles have invalid URL. In the output file, there is Article id, Article title, URL and info if the article is in DOAJ or not (articles in doaj are listed in the first place).

### CODE REVIEW

**Comment by amdomanska** [comment]
Please note that this script is quite slow as most of the job is run by python on the articles, not by ES query.

File: portality/scripts/articles_invalid_url.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "filter": {
+                "exists": {"field": "index.fulltext"}
+            },
+            "query": {
+                "match_all": {}
+            }
+        }
+    },
+    "sort": [{
+        "in_doaj": "desc"
```
Comment (Steven-Eardley) [inline portality/scripts/articles_invalid_url.py]: Does this do the ones that are in_doaj: true first? Do we want to do the ones that aren't in the DOAJ at all?

Comment (amdomanska) [reply portality/scripts/articles_invalid_url.py]: Yes, it does in_doaj: true first. And to be honest I don't know if we need one these that aren't in doaj at all so I just took all to check easily what is the scope of the issue.

**Comment by Steven-Eardley** [comment]
This is another PR that would have benefited from being a hotfix, but I'm happy to merge it to develop and it'll be ready to run after the next release.

**Comment by amdomanska** [comment]
Thank you!

**Pull request merged**